flink-libraries/flink-ml/pom.xml

```diff
@@ -116,6 +119,9 @@
 			<id>default</id>
 			<activation>
 				<activeByDefault>true</activeByDefault>
+				<property>
+					<name>default</name>
```

Comment:
This profile is never activated from the command-line and doesn't require the property.